src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/_params.py

```diff
@@ -27,12 +27,35 @@ def load_arguments(self, _):
         c.argument('enable_non_ssl_port', action='store_true')
         c.argument('shard_count', type=int)
 
+    with self.argument_context('redis import') as c:
```

Comment:
Instead of duplicating these entries, you should either:
1) use `self.argument_context('redis') as c:` OR
2) do something like:
```Python
for scope in ['redis import', 'redis import-method', 'redis export']:
   with self.argument_context(scope) as c:
       ...
``` 
OR
3) Create CLIArgumentTypes for files and format and use those.
```Python
format_type = CLIArgumentType(options_list=['--file-format'], help='...')

with self.argument_context('redis import') as c:
   c. argument('file_format', arg_type=format_type)
```